contracts/BoostedMasterChefJoe.sol

```diff
@@ -145,149 +170,101 @@ contract BoostedMasterChefJoe is Initializable, OwnableUpgradeable, ReentrancyGu
         emit Init(balance);
     }
 
-    /// @notice Returns the number of BMCJ pools.
-    /// @return pools The amount of pools in this farm
-    function poolLength() external view returns (uint256 pools) {
-        pools = poolInfo.length;
-    }
-
```

Comment:
I reckon we should leave it in

Reply:
It's still in L533, I'll move it because It's not in the `external` part of the contract